### PR TITLE
set prototype and fix ontrack

### PR DESCRIPTION
**Pull request by fippo**
had to do that to make #175 work. This should make the 'trackEvent.track is an object' messages appear in travis

### CODE REVIEW

**Comment by fippo** [comment]
also fixed the receivers-contain test which failed in Firefox after some discussion with @jan-ivar

File: adapter.js [hunk 1]
```diff
@@ -1676,7 +1678,8 @@ if (typeof window === 'undefined' || !window.navigator) {
 }
 
 // Polyfill ontrack on browsers that don't yet have it
-if (typeof window === 'object' && !('ontrack' in window.RTCPeerConnection)) {
+if (typeof window === 'object' && !('ontrack' in
+    window.RTCPeerConnection.prototype)) {
```
Comment (jan-ivar) [inline adapter.js:1682]: Should probably have a check in the middle here that window.RTCPeerConnection exists, just to be on the safe side?

**Comment by jan-ivar** [comment]
Thanks for this!

File: adapter.js (hunk 1 above)
Comment (KaptenJansson) [inline adapter.js:1682]: I agree.

**Comment by KaptenJansson** [comment]
LGTM apart from the comment from @jan-ivar.

File: adapter.js (hunk 1 above)
Comment (fippo) [inline adapter.js:1682]: had done that earlier but pushed in the wrong branch.

**Comment by fippo** [comment]
Before merging I want to fix that we currently require .ontrack to be set -- we should also support addEventListener('track', ...)

**Comment by jan-ivar** [comment]
@fippo can we do that in a separate PR? It's broken right now...

**Comment by fippo** [comment]
hrm. I would need to add a variable so... i'll do that in the modularized stuff.

**Comment by KaptenJansson** [comment]
Ok, so this is OK to merge then? The follow up PR will be done in after #164 ? I will try to land it this week.

**Comment by fippo** [comment]
can you bump npm and update gh-pages please?

**Comment by KaptenJansson** [comment]
Done

**Pull request merged**